cli/main.go

```diff
@@ -51,6 +54,60 @@ func main() {
 		EnableBashCompletion:   true,
 		UseShortOptionHandling: true,
 		Commands: []*cli.Command{
+			{
+				Name:  "assign-submitter",
+				Args:  true,
+				Usage: "Assign a different address to be able to submit your proofs. You'll always be able to submit from your EigenPod owner PK.",
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:        "json",
+						Value:       false,
+						Usage:       "print only plain JSON",
+						Required:    false,
+						Destination: &useJson,
+					},
+				},
+				Action: func(cctx *cli.Context) error {
+					targetAddress := cctx.Args().First()
+					if len(targetAddress) == 0 {
+						return fmt.Errorf("usage: `assign-submitter <0xsubmitter>")
+					}
```

Comment:
Why are we doing it like this instead of adding a flag like the other CLI commands?

Reply:
oh i see what you're going for. Problems with this:
- Try running this command without passing in a submitter address: it doesn't actually give you this message with usage text; it's too busy panicing about other required flags not being set.
- The help text for this command doesn't tell you how to use it either

Reply:
So here's a weird quirk -- if I run this command:

```
./cli status --podAddress $EIGENPOD_ADDRESS --beaconNode $NODE_BEACON --execNode $NODE_ETH
```

I get "panic: Required flags "podAddress, beaconNode, execNode" not set"

...but if I put `status` after all the flags, it works fine.

Is that normal? I feel like the order shouldn't matter

Reply:
im thinking a good way to address this and also fix some inconsistency with what flags are actually required would be to move some of these flags out of being "global required flags" and into specific commands

Reply:
One example of a thing this would fix is that technically the assign-submitter flag does not need a beacon RPC, but currently that's a globally-required flag.

Reply:
dw about it -- i fixed it in my refactor/PR

though if there was a better solution pls do tell me!